storage/integration_test.go

```diff
@@ -2212,6 +2211,10 @@ func TestIntegration_RequesterPays(t *testing.T) {
 	})
 
 	// ACL operations.
+	// Create another object for these to avoid object rate limits.
+	call("write object", func(b *BucketHandle) error {
+		return writeObject(ctx, b.Object("bar"), "text/plain", []byte("hello"))
```

Comment:
Do you think tests should start using a UUID as a suffix?

Reply:
In practice it hasn't been an issue except occasionally with this one test (and that's because this test needs to create an object and modify it several times quickly). I can add a UUID for each of foo and bar in this test though.

Reply:
Honestly this won't make any difference to the result since the test still needs to modify the same object multiple times; however using a UID still seems nice/good so I did it.